docs/source/general/management_commands.rst

```diff
@@ -42,3 +42,18 @@ For example:
 .. code-block:: shell
 
     ./manage.py delete_old_postauth 365
+
+``cleanup_stale_radacct``
+-------------------------
+
+This command closes active RADIUS sessions that are open since <days>.
```

Comment:
Please improve it as follows:

`This command closes stale RADIUS sessions that have remained open for the number of specified `<days>`.`